docs/01-app/02-guides/backend-for-frontend.mdx

```diff
@@ -403,6 +403,10 @@ Both provide methods for reading and manipulating cookies.
 
 You can pass `NextRequest` to any function expecting `Request`. Likewise, you can return `NextResponse` where a `Response` is expected.
 
+It is possible to set headers on a `Response` or `NextResponse` using `new Response(..., { headers })` or `NextResponse.json(..., { headers })`.
+
+Avoid echoing incoming request headers in responses. Middleware or libraries may append headers to incoming requests, and echoing these can leak sensitive information.
+
```

Comment:
```suggestion
```

I don't think this is necessary here, since it is mentioned further down and links back to the API for next